ambry-cloud/src/main/java/com.github.ambry.cloud/VcrReplicationManager.java

```diff
@@ -188,24 +168,27 @@ void addPartition(PartitionId partitionId) throws ReplicationException {
         FindTokenFactory findTokenFactory =
             tokenHelper.getFindTokenFactoryFromReplicaType(peerReplica.getReplicaType());
         RemoteReplicaInfo remoteReplicaInfo =
-            new RemoteReplicaInfo(peerReplica, cloudReplica, cloudStore, findTokenFactory.getNewFindToken(),
+            new RemoteReplicaInfo(peerReplica, cloudReplica, storeManager.getStore(partitionId),
```

Comment:
minor: save the result of `storeManager.getStore` to a variable and reuse it in this method.